beta/src/components/Layout/Sidebar/Sidebar.tsx

```diff
@@ -13,32 +13,44 @@ import {MobileNav} from '../Nav/MobileNav';
 import {Feedback} from '../Feedback';
 
 const SIDEBAR_BREAKPOINT = 1023;
+const SM_BREAKPOINT = 640;
 
 export function Sidebar() {
   const {menuRef, isOpen} = React.useContext(MenuContext);
   const isMobileSidebar = useMediaQuery(SIDEBAR_BREAKPOINT);
   let routeTree = React.useContext(SidebarContext);
   const isHidden = isMobileSidebar ? !isOpen : false;
 
+  const bannerHeight = useMediaQuery(SM_BREAKPOINT) ? 72 : 40;
+  let navMarginTop = bannerHeight;
+  if (typeof window !== 'undefined') {
+    const scrollAmount = document.documentElement.scrollTop;
```

Comment:
We shouldn't be reading global mutable values during render. This usually indicates a bug (even when it's not obvious how to trigger it).